src/highlevel.jl

```diff
@@ -452,3 +454,12 @@ macro roc(ex...)
         end
     return esc(code)
 end
+
+# launch config
+
+function launch_configuration(fun::Runtime.ROCFunction, device::ROCDevice; input_block_size=1, dynamic_localmem=0)
```

Comment:
Maybe we should get the `device` from `fun` as `fun.mod.exe.device`?
To avoid ambiguity...

```suggestion
function launch_configuration(fun::Runtime.ROCFunction; input_block_size=1, dynamic_localmem=0)
```